test/uk/gov/hmrc/helptosave/repo/MongoEmailStoreSpec.scala

```diff
@@ -39,143 +45,100 @@ class MongoEmailStoreSpec extends TestSupport with Eventually with MongoSupport
       .expects(input)
       .returning(output.fold[Try[String]](Failure(new Exception("uh oh")))(Success(_)))
 
-  def newMongoEmailStore(reactiveMongoComponent: ReactiveMongoComponent) =
-    new MongoEmailStore(reactiveMongoComponent, crypto, mockMetrics) {
-
-    }
+  def newMongoEmailStore(mongoComponent: MongoComponent) =
+    new MongoEmailStore(mongoComponent, crypto, mockMetrics)
 
   "The MongoEmailStore" when {
 
     val email = "EMAIL"
     val encryptedEmail = "ENCRYPTED"
 
-      def storeConfirmedEmail(nino: NINO, email: String, emailStore: MongoEmailStore): Either[String, Unit] =
-        await(emailStore.store(email, nino).value)
+      def storeConfirmedEmail(nino: NINO, email: String, emailStore: MongoEmailStore) =
+        emailStore.store(email, nino)
 
-      def getConfirmedEmail(nino: NINO, emailStore: MongoEmailStore): Either[String, Option[String]] =
-        await(emailStore.get(nino).value)
+      def getConfirmedEmail(nino: NINO, emailStore: MongoEmailStore) = emailStore.get(nino)
 
-      def deleteEmail(nino: NINO, emailStore: MongoEmailStore): Either[String, Unit] =
-        await(emailStore.delete(nino).value)
+      def deleteEmail(nino: NINO, emailStore: MongoEmailStore) = emailStore.delete(nino)
 
     "updating emails" must {
 
       "store the email in the mongo database" in {
         val nino = randomNINO()
-        val emailStore = newMongoEmailStore(reactiveMongoComponent)
+        val emailStore = repository
 
         inSequence {
           mockEncrypt(email)(encryptedEmail)
           mockDecrypt(encryptedEmail)(Some(email))
         }
 
-        storeConfirmedEmail(nino, email, emailStore)
-
-        val storedEmail = getConfirmedEmail(nino, emailStore)
-        storedEmail shouldBe Right(Some(email))
-
-      }
-
-      "return a right if the update is successful" in {
```

Comment:
These two test cases have been removed, were they not needed?

Reply:
yes they are no longer needed